src/sdk/namespace_fs.js

```diff
@@ -532,6 +547,17 @@ class NamespaceFS {
                 await this._upload_stream(params.source_stream, upload_path, fs_account_config);
             }
         } catch (err) {
+            try {
+                console.log('LMLM before upload_object report');
+                object_sdk.rpc_client.pool.update_issues_report({
+                    namespace_resource_id: this.namespace_resource_id,
+                    error_code: err.code,
+                    time: Date.now(),
+                });
+                console.log('LMLM after upload_object report');
+            } catch (e) {
+                console.log('update_issues_report on error:', e, 'ignoring.');
+            }
```

Comment:
1. Refactor to a function - we don't want to repeat this boilerplate everytime.
2. Lets filter to report only on errors with `err.code`